XML/TestCases/PerformanceTests.xml

```diff
@@ -748,4 +748,24 @@
         <Tags>lkp</Tags>
         <Priority>3</Priority>
     </test>
+    <test>
+        <testName>PERF-XDP-TCP-LATENCY</testName>
+        <testScript>PERF-XDP-TCP-LATENCY.ps1</testScript>
+        <setupType>TwoVM2NIC</setupType>
+        <OverrideVMSize>Standard_D64_v3</OverrideVMSize>
+        <AdditionalHWConfig>
+            <Networking>SRIOV</Networking>
+        </AdditionalHWConfig>
+        <files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\XDPDumpSetup.sh,.\Testscripts\Linux\perf_lagscope.sh</files>
+        <TestParameters>
+            <param>pingIteration=1000000</param>
```

Comment:
No need customization?

Reply:
I didn't get that.

Reply:
Can you put this number, 1000000 to the parameter like others?

Reply:
done